src/renderer/lib/state.js

```diff
@@ -69,10 +69,8 @@ function load (cb) {
   appConfig.read(function (err, saved) {
     if (err) {
       console.log('Missing config file: Creating new one')
-      onSavedState(null, {})
-    } else {
-      onSavedState(null, saved)
     }
+    onSavedState(null, err ? {} : saved)
```

Comment:
Now it's enough to call `onSavedState()` _once_, with a value depending on the value of `err`.